assignProdTaskChain.py

```diff
@@ -294,7 +306,24 @@ def main():
             if exist and procversion <= maxv:
                 print "Some output datasets exist, its advised to assign with v ==", maxv + 1
                 sys.exit(0)
-    
+
+        elif schema["RequestType"] == "Resubmission":
+            # For resubmission of a merge task inside a taskchain workflow, we cannot provide the acqera and procstring
+            if "Merge" in schema["InitialTaskPath"].split("/")[-1]:
+                acqera = None
+                procstring = None
+            # For another type of task, we need to look for the acqera and procstring information inside the
+            # original workflow
+            else:
+                if any("ProcessingString" in k for k, v in schema.items()):
```

Comment:
I'd replace it by something like `if "ProcessingString" in schema and "AcquisitionEra" in schema:`